requirements.txt

```diff
@@ -15,6 +15,7 @@ paramiko==3.4.0
 pillow==10.3.0
 psutil==5.9.8
 pycparser==2.22
+Pygments==2.18.0
```

Comment:
I think we should add the new dependency to pyinstaller as well

Reply:
I've tested pyinstaller and it works with this branch. So I think there's no such need.